packages/hyperliquid/.env.example

```diff
@@ -0,0 +1,7 @@
+# Monad Integration Test Environment Variables
+
+# Test mnemonic (12 or 24 words) - REQUIRED
+TEST_MONAD_MNEMONIC=your twelve word mnemonic phrase goes here for testing purposes only
```

Comment:
MONAD -> Hyperliquid (all over the place :))

Reply:
still a draft, will be removed